docs/root/configuration/listeners/listener_filters/original_dst_filter.rst

```diff
@@ -3,13 +3,42 @@
 Original Destination
 ====================
 
+Linux
+===============
+
 Original destination listener filter reads the SO_ORIGINAL_DST socket option set when a connection
 has been redirected by an iptables REDIRECT target, or by an iptables TPROXY target in combination
 with setting the listener's :ref:`transparent <envoy_v3_api_field_config.listener.v3.Listener.transparent>` option.
+
+Windows
```

Comment:
@pravb and @elweb9858 do you think we should elaborate more in Envoy docs?

Reply:
This looks good to me. IMO sidecar redirection should be covered by a blog post which could note the differences between the platforms.